validations/src/validations.rs

```diff
@@ -383,6 +383,7 @@ pub fn evaluate_tally_precondition_clause(
     reveals: Vec<RadonReport<RadonTypes>>,
     minimum_consensus: f64,
     num_commits: usize,
+    errors: &mut Vec<bool>,
```

Comment:
Any reason this is a separate argument and not part of `TallyPreconditionClauseResult`?

Reply:
good point